modules/speechInteraction/modules/googleSpeech/main.cpp

```diff
@@ -273,11 +273,6 @@ class Processing : public yarp::os::TypedReaderCallback<yarp::sig::Sound>
                 b.addString(alternative.transcript());
             }
         }
-        
-
-
-
-
         return b;
     }
 
```

Comment:
To avoid conflicts with the upcoming PR's, better off resetting this file outright.

Reply:
Resetted!